index.html

```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
+
+                    <div class="quality-card">
+                        <div class="icon-wrapper">
+                            <svg width="44" height="44" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <g clip-path="url(#clip0_34_2481)">
+                                <path d="M20.1996 4H29.1996C30.3996 4 31.1996 5.2 30.9996 6.4L26.5996 20.8H31.1996C32.5996 20.8 33.5996 22.2 32.9996 23.6L23.9996 42C23.5996 42.6 22.5996 42.4 22.7996 41.6L24.9996 27.6H17.9996C16.7996 27.6 15.9996 26.6 16.1996 25.4L18.3996 5.8C18.3996 4.8 19.1996 4 20.1996 4Z" fill="#36B37E"/>
+                                <path d="M2.43662 16.6706L5.64445 14.8185C6.07216 14.5716 6.60763 14.8406 6.78668 15.3153L8.22241 21.4238L9.86196 20.4772C10.361 20.1891 11.0094 20.4892 11.0876 21.1185L11.7183 29.6189C11.7009 29.9181 11.3027 30.0516 11.2071 29.7214L9.07068 24.2101L6.57571 25.6506C6.14799 25.8975 5.65424 25.7008 5.4752 25.2261L2.17055 17.6914C1.96194 17.33 2.08019 16.8764 2.43662 16.6706Z" fill="#091E42"/>
+                                <path d="M46.6256 16.6706L43.4178 14.8185C42.9901 14.5716 42.4546 14.8406 42.2756 15.3153L40.8398 21.4238L39.2003 20.4772C38.7013 20.1891 38.0528 20.4892 37.9746 21.1185L37.344 29.6189C37.3614 29.9181 37.7595 30.0516 37.8552 29.7214L39.9916 24.2101L42.4866 25.6506C42.9143 25.8975 43.408 25.7008 43.5871 25.2261L46.8917 17.6914C47.1003 17.33 46.9821 16.8764 46.6256 16.6706Z" fill="#091E42"/>
+                                </g>
+                                <defs>
+                                <clipPath id="clip0_34_2481">
+                                <rect width="48" height="48" fill="white"/>
+                                </clipPath>
+                                </defs>
+                                </svg>
+                                
+                        </div>
+                        <div class="quality-content">
+                            <div class="quality-name">Skills</div>
+                            <ul class="skill-tags">
+                                <li class="skill-tag">Codekit</li>
+                                <li class="skill-tag">GitHub</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                            </ul>
+                        </div>
+                        
+
+
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <div class="section contact-section">
+            <div class="container contact">
+                <div class="contact-info">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </div>
+
+                <div class="contact-form">
+                    <form class="form-block" action="submit">
```

Comment:
- [ ] Please make sure the `form` as an action and a method attribute that integrates with the formspree service.